### ISSUE TITLE
[Question] Missing: Status in Project View EAP2022.2 and 212.9.0

### ISSUE THREAD

**Issue by mrbscreen** [zielu/GitToolBox#405]
I am missing the numbers of commit on top of base and the Arrows in the project view:
![image](https://user-images.githubusercontent.com/7286722/175480754-33e1fb36-cd41-4eeb-88cc-3432087a885e.png)

As you can see, it is configured. Is there any way to get them back.

I am using:

* EAP2022.2
* GitToolbox 212.9.0

**Comment by zielu** [comment]
Hmm, I'm assuming that you do have changes that should be shown, right ? What happens if you re-open the project ? I've noticed that sometimes IDE fails to update the state so plugin does not get notification about new commits. You should be able to notice this in Git tool window too - no expected commits visible in history.

**Comment by jesselhk** [comment]
Could it be OS related? Mine hasn't been working in Windows but works in macOS and Linux. 
Local changes are picked up but not the remote.

**Comment by zielu** [comment]
There is OS dependant component to it - I'm reading repo state from built-in Git Integration APIs and afaik they are hooked into OS-dependant FS notifications. In latest version I added diagnostics mode to be able to inspect what plugin is seeing. I'll add instructions how to enable it later today.

**Comment by zielu** [comment]
@mrbscreen @jesselhk could you enable [diagnostic mode](https://github.com/zielu/GitToolBox/wiki/Manual#enable-diagnostic-mode) and check local branch hash vs parent branch hash. 
This information is taken from what built-in Git Integration is reporting and is not persisted by my plugin between restarts.

**Comment by abvm659** [comment]
Hi, I am experiencing the same issue. Any news about it? I am not sure when it started (most probably after upgrade to 2022.2.2). Having macOS Monterey. I don't see any error, anywhere in IDE. It says 
<img width="95" alt="image" src="https://user-images.githubusercontent.com/73121375/196125974-75aa30ea-de7e-44a5-a262-1747eac657d1.png">
But refresh, fetches, updates etc. all works fine. Using https git backends

**Comment by abvm659** [comment]
Ok got it .. needed to change status parent branch mode from `Auto` to `Tracked remote` - now it works fine (if someone will be dealing with the same issue)

<img width="464" alt="image" src="https://user-images.githubusercontent.com/73121375/196136037-9831a409-1fa0-4b91-a86f-60b43cf17ead.png">

**Comment by zielu** [comment]
Hi, I'll see what might be wrong - auto should be a good default.

**Comment by abvm659** [comment]
Hi, no I am not using youtrack task server, just plain default task management plugin if it is what you are asking for? The only thing that might be related I am using is the settings repository (local directory with auto sync)

**Comment by zielu** [comment]
Yes, I meant default task management. I'll try to reproduce this.
